samples/meetings-details-tab/nodejs/server/api/homeController.js

```diff
@@ -0,0 +1,40 @@
+const configuration = require('dotenv').config();
+const store = require('../services/store')
+const {createAdaptiveCard }= require('../services/AdaptiveCardService')
+const env = configuration.parsed;
+const appID = env.BotId;
+const appPassword = env.BotPassword;
+const { ConnectorClient, MicrosoftAppCredentials } = require('botframework-connector');
+const credentials = new MicrosoftAppCredentials(appID,appPassword);
+
+ const sendAgenda = async (req) => {
+    const data = req.body;
+    store.setItem("agendaList", data.taskList);
+    const ConversationID = store.getItem("conversationId");
+    const ServiceUrl = store.getItem("serviceUrl");
+    const client = new ConnectorClient(credentials, { baseUri: ServiceUrl });
```

Comment:
`baseUri` it is being set as service URL not as base URL please rename this to serviceURL